components/layout/Header.tsx

```diff
@@ -0,0 +1,54 @@
+import Logo from "@/public/images/logo/logo.svg";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+import {
+  GlobalHeader,
+  HeaderLeft,
+  HeaderLogo,
+  LoginLink,
+  NavItem,
+  NavList,
+} from "./Header.styles";
+
+function getLinkStyle(isActive: boolean) {
+  return { color: isActive ? "var(--blue)" : undefined };
+}
```

Comment:
P3:
해당 함수가 실행되면 `CSSProperties`를 반환하니 아래처럼 반환값을 명시해주시는 것이 더 좋을 것 같습니다.

```suggestion
function getLinkStyle(isActive: boolean): CSSProperties['color'] {
  return { color: isActive ? "var(--blue)" : undefined };
}
```